nilearn/version.py

```diff
@@ -1 +1,91 @@
+# *- encoding: utf-8 -*-
+"""
+nilearn version, required package versions, and utilities for checking
+"""
+# Author: Alexandre Abraham, Philippe Gervais, Ben Cipollini
+# License: simplified BSD
+
 __version__ = '0.1b1'
+
+# All metadata needed for required modules:
+#    * minver: minimum required version of the module
+#    * manual: whether the module must be installed manually
+# See more info in check_module_dependencies docstring.
+#
+# This is a list to preserve order, and to avoid the Python 2.7
+#   dependency of collections.OrderedDict
+REQUIRED_MODULE_METADATA = (
+    ('numpy', {
+        'minver': '1.6.0',
+        'manual_install': True,
+        'install_info': 'Install via your OS package manager (Linux) or Anaconda (OSX, Windows).'}),
```

Comment:
I would point to the installation instructions on the website: http://nilearn.github.io/introduction.html#installation